torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -509,8 +509,8 @@ Tensor TensorExprKernel::computeValue(const torch::jit::Value* v) {
 
   auto outputType = findDtypeForValue(v);
   std::vector<ExprHandle> outputShape = sizesForValue(v);
-  std::vector<ExprHandle> outputStrides;
-  if (is_channels_last_contiguous && (!is_contiguous)) {
+  std::vector<ExprHandle> outputStrides = {};
+  if (memory_layout_policy_ == MemoryLayoutPolicy::kChannelsLastNdContiguous) {
```

Comment:
It looks like the loop above is not needed anymore (where we compute `is_contiguous` and `is_channels_last_contiguous`). Could you please remove it?

Reply:
Oh, yes. Removed.